source/_integrations/input_datetime.markdown

```diff
@@ -4,6 +4,7 @@ description: Instructions on how to integrate the Input Datetime integration int
 ha_category:
   - Automation
 ha_release: 0.55
+ha_iot_class: Calculated
```

Comment:
This integration does not calculate a state, nor does it get one from a device.

```suggestion
ha_iot_class: ~
```

This will set the IoT class to none/null explicitly